dcrjson/walletsvrcmds.go

```diff
@@ -524,6 +524,17 @@ func NewSignRawTransactionCmd(hexEncodedTx string, inputs *[]RawTxInput, privKey
 	}
 }
 
+// VerifySeedCmd defines the walletlock JSON-RPC command.
+type VerifySeedCmd struct {
+	Seed string
+}
```

Comment:
we should add a second, optional, parameter so that we can verifyseed on a watching only wallet with an imported account xpub.  the parameter is needed to specify which account key to derive from the seed to check if it matches the current account (since it may not always be account 0).